tests/integration-tests-utils/src/main/java/org/apache/pulsar/tests/DockerUtils.java

```diff
@@ -61,11 +62,13 @@ private static File getTargetDirectory(String containerId) {
         return directory;
     }
 
-    public static void dumpContainerLogToTarget(DockerClient docker, String containerId) {
-        File output = new File(getTargetDirectory(containerId), "docker.log");
+    public static void dumpContainerLogToTarget(DockerClient dockerClient, String containerId) {
+        InspectContainerResponse inspectContainerResponse = dockerClient.inspectContainerCmd(containerId).exec();
+        final String containerName = inspectContainerResponse.getName().replace("/","");
```

Comment:
make a comment why `replace("/", "")`

Reply:
Updated description with motivation.

Reply:
please make a comment why `replace("/", "")` here.

Reply:
we remove the unnecessary forward slash in InspectContainerResponse to be more consistent with docker ps output.

Reply:
understand that. a comment inline would be helpful for people reading this code.